aegis_gym/rsl/grasp_train.py

```diff
@@ -0,0 +1,234 @@
+import argparse
+import re
+import pickle
+from importlib import metadata
+from pathlib import Path
+
+try:
+    try:
+        if metadata.version("rsl-rl"):
+            raise ImportError
+    except metadata.PackageNotFoundError:
+        if metadata.version("rsl-rl-lib") != "2.2.4":
+            raise ImportError
+except (metadata.PackageNotFoundError, ImportError) as e:
+    raise ImportError(
+        "Please uninstall 'rsl_rl' and install 'rsl-rl-lib==2.2.4'."
+    ) from e
+
+from rsl_rl.runners import OnPolicyRunner
+from behavior_cloning import BehaviorCloning
+
+import genesis as gs
```

Comment:
Does the order matter? If not, why not:

```suggestion
import genesis as gs
from rsl_rl.runners import OnPolicyRunner
from behavior_cloning import BehaviorCloning
```